resources/ext/cli/dropsonde.erb

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+PUPPET_AGENT_RUBY="/opt/puppetlabs/puppet/bin/ruby"
+DROPSONDE_DIR="/opt/puppetlabs/server/data/puppetserver/dropsonde"
+DROPSONDE_BIN="${DROPSONDE_DIR}/bin/dropsonde"
+
+export GEM_HOME=${DROPSONDE_DIR}
+export GEM_PATH=${DROPSONDE_DIR}
+
+if [[ -f "$DROPSONDE_BIN" && -x "$DROPSONDE_BIN" ]]
+then
+    $DROPSONDE_BIN "$@"
```

Comment:
I think this might need to have `PUPPET_AGENT_RUBY` in front of it. Otherwise it will try to run under JRuby due to a quirk of how it was installed.

Reply:
good catch, that's what I meant to do